tests/draft2019-09/optional/ecmascript-regex.json

```diff
@@ -1,11 +1,38 @@
 [
     {
         "description": "ECMA 262 regex non-compliance",
-        "schema": { "format": "regex" },
```

Comment:
Can we leave this original test in place as is as well? (And change valid to true if that's the case)?

Reply:
Yes, I'll put that back.